simple-repository-API/Database_Connection.cs

```diff
@@ -16,10 +16,11 @@ public List<Student> Database_connect()
                    @"Server = Laptop-SS4D3ECJ\SQLEXPRESS; Database = school; Trusted_Connection = True;"))
             {
                 connection.Open();
-                string queryString = "SELECT ID_Student,Name,Surname,Age FROM Student";
+                string queryString = "SELECT ID_Student,Name,Surname,Age FROM Student WHERE ID_Student = 1";
```

Comment:
To nie ma prawa działać. Teraz pobierasz tylko 1 studenta. Czyli jak wyżej podam np id student jako 5 to i tak pobierze 1